### PR TITLE
Added call to participate button in Kyverno Certification 

### PR DESCRIPTION
**Pull request by Anita-ihuman**
This PR is a fix for issue #302 
Added a call to action for Kyverno Certification and Adopters program.

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -11,6 +11,9 @@ <h1 class="text-center">Kubernetes Native Policy Management</h1>
 	</a>
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="/docs/introduction/#quick-start">
 		Get Started <i class="fa fa-arrow-alt-circle-right ml-2 "></i>
+	  </a>
+	  <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://learn.nirmata.com/explore">
+		Get Certified <i class="fa fa-certificate ml-2 "></i>
```
Comment (JimBugwadia) [inline content/en/_index.html:16]: Hi Anita, I get the intent of showcasing this, but don't think this button is appropriate in this location. 

What I suggest is that we either add a "Support" page (like here https://www.openpolicyagent.org/support) and list the certification, and other offers there. What do you think?

File: content/en/Community/_index.md [hunk 2]
```diff
@@ -43,3 +43,10 @@ We love hearing from our community!
...
 
 The project contributors use [GitHub discussions](https://github.com/kyverno/kyverno/discussions) for all implementation and project internal topics and the [GitHub Wiki](https://github.com/kyverno/kyverno/wiki) for design documents, notes on the dev environment, project guidelines and best practices.
+
+## Join Kyverno Adopters
+Kyverno is currently a CNCF sandbox project and as a requirement by the CNCF, every sandbox project requires a minimum of three adopters along with other specifications in order graduate to the next maturity level [incubation level](https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc).
+ 
+The goal of this adopters program is to take Kyverno from a [CNCF](https://www.cncf.io/) sandbox project to a CNCF incubator project. We hope to achieve this by getting the existing and new adopters of kyverno to share their stories with CNCF. Adopters who join this project will be added to the [Adopters List](https://github.com/kyverno/kyverno/blob/main/ADOPTERS.md).
```
[the remark this answers is not in the document]
Comment (JimBugwadia) [reply content/en/Community/_index.md:50]: Based on recent discussions, we learnt that the ToC can do private reference checks. So, would suggest we change this to:


> Kyverno is currently a CNCF sandbox project is growing fast. To qualify for the [incubation level](https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc) the CNCF requires production usage by multiple end users.

> The goal of this adopters program is to gather real-world usage examples and help Kyverno achieve the incubating state criteria. We hope to achieve this by learning from the existing and new adopters of Kyverno who are willing to share their stories with CNCF. The adopters can remain private or choose to be listed in the public [Adopters List](https://github.com/kyverno/kyverno/blob/main/ADOPTERS.md).

File: content/en/Community/_index.md [hunk 3]
```diff
@@ -43,3 +43,10 @@ We love hearing from our community!
...
+ 
+To participate, kindly fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeMJxbK6eQ-MpsbaXZ3GKp8NEz9F932keL2pwznI1JIfi7Iqw/viewform) (It will take 5 mins of your time).
```
Comment (JimBugwadia) [inline content/en/Community/_index.md:52]: Can we add a field to the form to check if the user wants their story to be included in the public adopters list?

**Review by JimBugwadia — CHANGES_REQUESTED** [review]

File: content/en/Community/_index.md (hunk 3 above)
Comment (Anita-ihuman) [reply content/en/Community/_index.md:52]: Yes I will include that

File: content/en/_index.html (hunk 1 above)
Comment (Anita-ihuman) [reply content/en/_index.html:16]: I think it is a great idea but I doubt support should be the name.

Comment (Anita-ihuman) [reply content/en/_index.html]: If that's the case tho,  should the file be exiting under community?
or should it be a stand-alone file?

Comment (Anita-ihuman) [reply content/en/_index.html]: It can also be a header under the resources tho

Comment (chipzoller) [reply content/en/_index.html]: We could create a "Training and Certification" section under the Community page and link to this plus Nirmata's [training course](https://learn.nirmata.com/courses/kyverno-in-production).

Comment (JimBugwadia) [reply content/en/_index.html]: Seems like we should create an area where other companies can also create PRs to list their commercial offers, distributions, trainings, services, etc. for Kyverno. 

If not "support" what would be a good short name for such a section?

Comment (chipzoller) [reply content/en/_index.html]: "Commercialization"?

Comment (Anita-ihuman) [reply content/en/_index.html]: How about "Services"?

Comment (JimBugwadia) [reply content/en/_index.html]: May be worth checking with the CNCF on this. Will check if Kim has more information.

@Anita-ihuman - if you can split out the adopters program updates, we can merge those changes and use a separate PR for the certification changes.

Comment (Anita-ihuman) [reply content/en/_index.html]: alright then will do that.

**Comment by Anita-ihuman** [comment]
@chipzoller I have deleted the certificated button. So it's just the adopter's program here

**Review by chipzoller — APPROVED** [review]

**Review by chipzoller — APPROVED** [review]

**Review by JimBugwadia — APPROVED** [review]

**Pull request merged**